pkg/skaffold/schema/validation/validation.go

```diff
@@ -99,6 +101,18 @@ func ProcessToErrorWithLocation(configs parser.SkaffoldConfigSet, validateConfig
 	return errs
 }
 
+func validateKptDeployerVersion(cfg *parser.SkaffoldConfigEntry, dc latest.DeployConfig) (cfgErrs []ErrorWithLocation) {
+	if dc.KptDeploy != nil {
```

Comment:
nit: invert conditional
```
if dc.KptDeploy == nil {
  return
}
...
```

Reply:
I changed it to do an early return, thanks! Done.